lib/data/seed/post-2020-timeline-2019-06-12.csv

```diff
@@ -77,17 +77,19 @@ Regional Consultation on the Post-2020 Global Biodiversity Framework for Central
 Towards a Global Pact for the Environment: Third Substantive Session,20/05/2019,23/05/2019,,"Nairobi, Kenya",UN Environment,This meeting was the third in a series which considered a technical and evidence-based report that identifies and assesses possible gaps in international environmental law and environment-related instruments with a view to strengthening their implementation.,It is expected environmental laws and their enforcement will play a role in strengthening the post-2020 agenda and meeting its aims and goals. ,http://sdg.iisd.org/events/towards-a-global-pact-for-the-environment-third-substantive-session/,Intergovernmental processes,Other relevance
 International Day for Biological Diversity 2019,22/05/2019,22/05/2019,,Worldwide,United Nations,"This day was aimed at increasing an understanding and awareness of biodiversity issues. The theme of this year is, ""Our Biodiversity, Our Food, Our Health”.",,https://www.cbd.int/idb/2019/,Communications,Other relevance
 "Conference on ""Biodiversity and Ecosystem Services""",23/5/2019,24/5/2019,,"Brussels, Belgium",The European Commission,This meeting took stock of progress towards the EU nature and biodiversity objectives as well as launching the public consultation for the final evaluation of the EU Biodiversity Strategy. There will also be a series of technical discussions with stakeholders on the evaluation of the EU biodiversity strategy to 2020 and potential EU contributions to the post-2020 global biodiversity framework. ,This meeting enabled discussions on how the EU will contribute towards the post-2020 agenda.,,,Direct contribution
-CITES COP18,23/05/2019,03/06/2019,x,"Colombo, Sri Lanka",CITES Secretariat,The eighteenth meeting of the Conference of the Parties to the Convention on International Trade in Endangered Species of Wild Fauna and Flora (CITES COP18) is expected to address a range of issues ranging from implementation of the Convention and species-specific matters to trade controls and the listing of new species in the Appendices to the CITES.,"Resolutions, decisions and intentions coming out of the meeting will likely form an important component of how national and international strategies on wildlife trade are shaped over the coming years, thus representing relevant input to the post-2020 biodiversity agenda.",https://cites.org/eng/news/the-secretary-generals-statement-on-the-postponement-of-cop18_26042019,Intergovernmental processes,Other relevance

```

Comment:
Just double checking, has this event in Sri Lanka been removed completely?
Can't remember we also consider event deletion 🤔

Reply:
Thanks @Levia I have just emailed Holly about this.